### PR TITLE
feat: Add support for `Window.Title` in Uno.WinUI

### PR DESCRIPTION
**Pull request by MartinZikmund**
GitHub Issue (If applicable): closes #

## PR Type

What kind of change does this PR introduce?

- Feature

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: src/Uno.UI/UI/Xaml/Window.WinAppSdk.cs [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+	/// Gets or sets a string used for the window title.
+	/// </summary>
+	public string Title
+	{
+		get => ApplicationView.GetForCurrentView().Title;
+		set => ApplicationView.GetForCurrentView().Title = value;
+	}
+#endif
+}
+#endif
```
Comment (Youssef1313) [inline src/Uno.UI/UI/Xaml/Window.WinAppSdk.cs:21]: Why not `#if HAS_UNO_WINUI && !__IOS__` together at the beginning?

Comment (MartinZikmund) [reply src/Uno.UI/UI/Xaml/Window.WinAppSdk.cs:21]: There will be more "WinAppSDK-specific" code for `Window`, so to make future changes easier, the file itself is Uno.WinUI, but just the property is not supported on iOS yet.

**Review by Youssef1313 — APPROVED** [review]

**Pull request merged**